### PR TITLE
[T2A2][W10-A4]Chen Penghao

### PR DESCRIPTION
**Pull request by chenpenghao**

### CODE REVIEW

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -159,6 +159,20 @@
...
     private static final char INPUT_COMMENT_MARKER = '#';
+    
+    /**
+     * length of input in processProgramArgs. If length of input is greater than 2, then it is invalid;
+     * if length of input is 1, then setup a given file for storage
+     * if length of input is 0, then setup a default file
+     */
+    private static final int LENGTHINVALID = 2;
+    private static final int LENGTHSETUPGIVENFILE = 1;
+    private static final int LENGTHSETUPDEFAULT = 0;
```
Comment (hwkchia) [inline src/seedu/addressbook/AddressBook.java:170]: Suggest using underscore as separator for constants?

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -181,7 +195,7 @@
     /**
      * List of all persons in the address book.
      */
-    private static final ArrayList<String[]> ALL_PERSONS = new ArrayList<>();
+    private static final ArrayList<String[]> ALL_PERSONS = new ArrayList<String[]>();
```
Comment (hwkchia) [inline src/seedu/addressbook/AddressBook.java:198]: The Java 8 standard does not require `ArrayList<String[]>`

File: src/seedu/addressbook/AddressBook.java [hunk 3]
```diff
@@ -256,17 +270,18 @@ private static void echoUserCommand(String userCommand) {
...
             exitProgram();
         }
 
-        if (args.length == 1) {
+        if (args.length == LENGTHSETUPGIVENFILE) {
             setupGivenFileForStorage(args[0]);
         }
 
-        if(args.length == 0) {
+        if(args.length == LENGTHSETUPDEFAULT) {
```
Comment (hwkchia) [inline src/seedu/addressbook/AddressBook.java:284]: Thumbs up for dealing with magic numbers

File: src/seedu/addressbook/AddressBook.java [hunk 4]
```diff
@@ -793,7 +817,7 @@ private static void addPersonToAddressBook(String[] person) {
      * @param exactPerson the actual person inside the address book (exactPerson == the person to delete in the full list)
      * @return true if the given person was found and deleted in the model
      */
-    private static boolean deletePersonFromAddressBook(String[] exactPerson) {
+    private static boolean hasDeletePersonFromAddressBook(String[] exactPerson) {
```
Comment (hwkchia) [inline src/seedu/addressbook/AddressBook.java:820]: Thumbs up for boolean naming

**Review by hwkchia — COMMENTED** [review]
Good effort in applying coding standards.  Maintain this level in all future work.

**Comment by hwkchia** [comment]
@xmchenph Some comments added. Please close the PR after reading comments.

File: src/seedu/addressbook/AddressBook.java (hunk 1 above)
Comment (chenpenghao) [reply src/seedu/addressbook/AddressBook.java:170]: should be "LENGTH_INVALID", "LENGTH_SET_UP_GIVEN_FILE" and "LENGTH_SET_UP_DEFAULT"

**Pull request closed without merging**